test/apps/iodemo/run_io_demo.sh

```diff
@@ -464,8 +478,21 @@ kill_all() {
 }
 
 run() {
-	trap kill_all SIGINT
-	${launcher} ${host_list} "$PWD/iodemo_commands_\$(hostname -s).sh"
+	if [ ${dry_run} -eq 1 ]
```

Comment:
now it prints generated script on stdout which is redundant IMO
maybe just generate script and exit?
if you need "print script on stdout" functionality - it can be additional option?

Reply:
it's done only in verbose mode (see line 487)